lib/jwt_sessions/store_adapters/abstract_store_adapter.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'redis'
+
+module JWTSessions
+  module StoreAdapters
+    def self.build_by_name(adapter, options = {})
```

Comment:
Let's extract it into `lib/jws_sessions/store_adapters`; this logic doesn't belong to abstract adapter.

We can also move all the `require 'jwt_sessions/store_adapters/*_store'` lines from `jwt_sessions.rb` there.

Reply:
👍